sdk/core/src/CommonNetworkMember/BaseNetworkMember/BaseNetworkMember.ts

```diff
@@ -1,11 +1,4 @@
-import {
-  DidDocumentService,
-  JwtService,
-  KeysService,
-  MetricsService,
-  Affinity,
-  generateFullSeed,
-} from '@affinidi/common'
+import { DidDocumentService, DidResolver, JwtService, KeysService, MetricsService, Affinity } from '@affinidi/common'
```

Comment:
What is the point of moving this file? `CommonNetworkMember` directory was created specifically for `BaseNetworkMember` and its descendants.

Reply:
Due to I ope in the future we can refactor this class with 1100 lines of code, so it makes sense to keep all it parts in one place